.mvn/jvm.config

```diff
@@ -11,3 +11,5 @@
 --add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED
 -XX:+UnlockDiagnosticVMOptions
 -XX:GCLockerRetryAllocationCount=100
+-Daether.syncContext.named.factory=file-lock
```

Comment:
These should go into `.mvn/maven.config` instead.

Reply:
we set the MAVEN_CONFIG environment variable already in the CI. Will this merge with the .`mvn/maven.config`?